ui/components/Filters.js

```diff
@@ -334,44 +335,6 @@ function MesheryFilters({
    * publish filter capability and setting the canPublishFilter state accordingly
    */
   useEffect(() => {
-    dataFetch(
-      '/api/schema/resource/filter',
-      {
-        method: 'GET',
-        credentials: 'include',
-      },
-      (result) => {
-        setImportSchema(result);
-      },
-      handleError(ACTION_TYPES.SCHEMA_FETCH),
-    );
-    dataFetch(
-      '/api/schema/resource/publish',
-      {
-        method: 'GET',
-        credentials: 'include',
-      },
-      async (result) => {
-        try {
-          const { models } = await getMeshModels();
-          const modelNames = _.uniq(models?.map((model) => model.displayName));
-          modelNames.sort();
-
-          // Modify the schema using the utility function
-          const modifiedSchema = modifyRJSFSchema(
-            result.rjsfSchema,
-            'properties.compatibility.items.enum',
-            modelNames,
-          );
-          setPublishSchema({ rjsfSchema: modifiedSchema, uiSchema: result.uiSchema });
-          setMeshModels(models);
```

Comment:
@11happy thanks for removing this code, also it seems like we also removed code for "setMeshModels" here, by default publish schema doesn't have meshmodels, for that we have this function where we fetch meshmodels and update schema `properties.compatibility.items.enum` property with those meshmodels, as we are now fetching it directly from sistent this will cause issue and publish modal will not show any dropdown in technology field becuse meshmodels is not present in publish schema

Can you make sure for publish schema we update meshmodels, you can use this function to update meshmodels to schema property

```

          // Modify the schema using the utility function
          const modifiedSchema = modifyRJSFSchema(
            result.rjsfSchema,
            'properties.compatibility.items.enum',
            modelNames,
          );
```

As you can see in below shema link, in this enum field only kubernetes is present by default, however there should be 190 meshmodels, and to make sure they are dynamic in nature and we don't need to update schema again and again we update publish schema in Meshery UI, so can you make sure that we don't remove that logic.

https://github.com/layer5io/sistent/blob/master/src/schemas/publishCatalogItem/schema.tsx#L12

You can have a test case that

- [ ] Changes made in migrating schema from API to Sistent in Meshery UI should not affect the UI

Reply:
done